src/Organizations/OrganizationDetails/OrganizationDetailsContainer.js

```diff
@@ -70,6 +70,17 @@ const OrganizationDetailsContainer = ({
     [location.search],
   );
 
+  const editOrganization = useCallback(
+    () => {
+      history.push({
+        pathname: `/organizations/new_view/${organizationId}/edit`,
+        search: location.search,
```

Comment:
is there any reason to do so instead of history.push(`/receiving/${id}/view`) ?

Reply:
to keep filter settings